### ISSUE TITLE
Godot 3.3 2D physics add_torque

### ISSUE THREAD

**Issue by chrisskeddy** [godotengine/godot#48857]
Godot version: 3.3

**OS/device including version:** Windows 10 64bit GLES 3, GTX 960

**Issue description:**
add_torque() for RigidBody2D never stops spinning

Press left/right key. It seems it is needed to press them 6 times to get it moving.
Also you still need to press the key 3 times even if the values are changed from 10 to 100

**Minimal reproduction project:**
https://github.com/chrisskeddy/godot_2dphysics

**Comment by pouleyKetchoupp** [comment]
This is due to the fact 2D physics doesn't clear forces and torques on each frame when `add_force()` and `add_torque()` are called. These functions add a constant force that keeps being applied, and they can only be cleared using `applied_force = Vector2()` and `applied_torque = 0.0`.

It was reported in #38646, and it will be fixed by making things consistent in 2D and 3D, as well as making the documentation more clear.

What you can do to set a one time rotation is to use `apply_torque_impulse()` instead of `add_torque()`. Impulses are applied only once, then the rotation will stop more or less fast depending on the `angular_damp` value set on the rigid body (or in project settings for all rigid bodies).

So your script can be changed to something like that:

```
func _process(delta):
	if Input.is_action_just_pressed("ui_right"):
		$RigidBody2D.apply_torque_impulse(5000 * delta)
	elif Input.is_action_just_pressed("ui_left"):
		$RigidBody2D.apply_torque_impulse(-5000 * delta)
```

Note that impulses use a different unit than forces, so you need to multiply by `delta`, but the value also needs to be higher because it's applied only for one frame.

---

I'm closing this issue as duplicate of #38646, but feel free to ask more questions if needed.
